modules/appeals_api/app/models/appeals_api/appellant.rb

```diff
@@ -129,10 +142,34 @@ def ssn_last_four
       ssn&.last(4)
     end
 
+    def timezone
+      form_data&.dig('timezone').presence&.strip
+    end
+
+    def submission_email_identifier
+      email_id = "#{type.downcase}_email"
+
+      return { id_type: email_id, id_value: email } if email.present?
```

Comment:
Could you double check how `id_type` is being used in the email notification handling and ensure that having a an `id_type: "claimant_email"` will be ok w/ VANotify (assuming we even send this data to them)?

Reply:
Yes I'll look into that now! I thought the id_type was just for us, but I'm going to make sure.